plugins/inputs/ethtool/namespace_linux.go

```diff
@@ -36,7 +36,7 @@ func (n *NamespaceGoroutine) Interfaces() ([]NamespacedInterface, error) {
 	interfaces, err := n.Do(func(n *NamespaceGoroutine) (interface{}, error) {
 		interfaces, err := net.Interfaces()
 		if err != nil {
-			n.Log.Errorf(`Could not get interfaces in namespace "%s": %s`, n.name, err)
+			n.Log.Errorf("Could not get interfaces in namespace %q: %w", n.name, err)
```

Comment:
`%w` should be only used for creating error, not for printing. Use `%s` or `%v` instead.

Reply:
Not sure if in this case this `n.Log` statement is required, since the `err` is returned anyway.. WDYT?